### DISCUSSION TITLE
Open to new flavours?

### DISCUSSION THREAD

**Discussion by linden-dg** [catppuccin/catppuccin#2356] [Ideas]
I really love this set of themes, but I need a bit more contrast than is provided by e.g. the Mocha flavour. I've been working on a variant from Mocha with a darker background etc. 

Are you open new variants/flavours, or would you prefer I work on it as a separate theme?

**Comment by nullishamy** [comment]
Hey there,

We don't want to support more than our 4 flavours 'officially' at this time, but many of our popular ports (e.g neovim / vscode) support customisation of the palette, so you could add your own twist there!

Our neovim port has an open issue with such community made variants, and even includes the unofficial 'OLED' flavor, which seem to be along the lines of what you're looking for? More details here: https://github.com/catppuccin/nvim/discussions/323#discussion-4496786 

This may change in the future, as we become more able to automatically generate our ports rather than hand rolling every single one, but this isn't on the cards for now!
